template/package.json

```diff
@@ -10,7 +10,7 @@
     "unit": "karma start test/unit/karma.conf.js --single-run"{{/unit}}{{#e2e}},
     "e2e": "node test/e2e/runner.js"{{/e2e}}{{#if_or unit e2e}},
     "test": "{{#unit}}npm run unit{{/unit}}{{#unit}}{{#e2e}} && {{/e2e}}{{/unit}}{{#e2e}}npm run e2e{{/e2e}}"{{/if_or}}{{#lint}},
-    "lint": "eslint --ext .js,.vue src{{#unit}} test/unit/specs{{/unit}}{{#e2e}} test/e2e/specs{{/e2e}}"{{/lint}}
+    "lint": "eslint --ext .js,.vue src{{#unit}} test/unit/specs{{/unit}}{{#e2e}} test/e2e/specs{{/e2e}}; exit 0"{{/lint}}
```

Comment:
This can be a bit dangerous actually, as any tasks relying on the exit code from `npm run lint` would have it suppressed. Users would have to look very carefully to see that this was happening. I think what you actually want is `npm run lint -s`, which silences NPM's useless error messages. Or, you can run these with yarn: `yarn run` produces only 2 additional lines when a command fails:

```
error Command failed with exit code 1.
info Visit http://yarnpkg.com/en/docs/cli/run for documentation about this command.
```